### PR TITLE
Saving and updating cgh, hpos and indic

### PR DESCRIPTION
**Pull request by FrancisRenaud**
This does not include adding or deleting hpos yet.

### CODE REVIEW

**Review by YanLabCHU — CHANGES_REQUESTED** [review]
Some more comments:

- remove `hpo_terms.json` if it's not used

- Don't delete `pnpm-lock.yaml` from the repository

File: client/src/components/screens/PatientSubmission/ClinicalInformation.jsx [hunk 1]
```diff
@@ -11,6 +12,152 @@ import {
   ic_add, ic_remove, ic_help, ic_person, ic_visibility, ic_visibility_off,
 } from 'react-icons-kit/md';
 
+import {
+  CGH_CODES,
+  CGH_VALUES,
+  isCGH,
+  isIndication,
+  cghInterpretation,
```
[the remark this answers is not in the document]
Comment (FrancisRenaud) [reply client/src/components/screens/PatientSubmission/ClinicalInformation.jsx]: First 3 are used. Last one has been removed in cleanup.

[the remark this answers is not in the document]
Comment (YanLabCHU) [reply client/src/components/screens/PatientSubmission/ClinicalInformation.jsx]: Yeah, I was talking about that one, Github shows 4 lines for some reason lol

**Review by YanLabCHU — CHANGES_REQUESTED** [review]
Like said before: 
- Remove `hpo_terms.json` if it's not used
- Don't delete `pnpm-lock.yaml` from the repository

**Comment by FrancisRenaud** [comment]
This PR is now obsolete

**Pull request closed without merging**